packages/client/src/object/object.test.ts

```diff
@@ -57,11 +57,11 @@ describe("OsdkObject", () => {
 
       // we should get the employee we requested
       const employee = result.data[0];
+      expect(JSON.stringify(employee)).toBeDefined();
```

Comment:
Deleted title property from this object and making sure stringify works (this is the function that failed for our consumers).

Reply:
Also added tests above in createOsdkObject.test